spec/models/configuration_script_base_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe ConfigurationScriptBase do
+  it "has many authentications and vice versa" do
```

Comment:
> I am simply asserting the relationship to be maintained.

In that case this *really* is testing Rails associations and doesn't need to, IMO ^_^

Testing that things are associated with other things is basically saying "does Rails do it's job when I put `has_many :configuration_script_bases` in my code?" We presume, that using the Rails API (which is tested separately) does indeed do this thing. The only test coverage this covers is if someone later on down the road accidentally deletes the line with the association, which should be something that we presume would be noticed very quickly (as it would break everything via other tests) and caught in code review besides.

This explanation is why I'm against using the [shoulda-matchers gem](https://github.com/thoughtbot/shoulda-matchers) Look at one of them as example: 

> `validate_confirmation_of` tests usage of validates_confirmation_of

Since we assume `validates_confirmation_of` *works* (because it's Rails' concern), this literally tests 'is the line of code I wrote present?' That's not very helpful.

I hope that makes sense. If you *really* disagree and insist upon having some sort of mechanism to test, you should at least initialize this models and not create db records for them. The associations you're looking to ensure work in-memory on assignment and don't necessarily need to be written to be seen.